templates/zombienet/tests/smoke.rs

```diff
@@ -7,28 +7,73 @@
 //! `cargo build --package minimal-template-node --release`
 //! `export PATH=<path-to-polkadot-sdk-repo>/target/release:$PATH
 //!
-//! The you can run the test with
-//! `cargo test -p template-zombienet-tests`
+//! There are also some tests related to omni node which run basaed on pre-generated chain specs,
+//! so to be able to run them you would need to generate the right chain spec (just minimal and
+//! parachain tests supported for now).
+//!
+//! You can run the following command to generate a minimal chainspec, once the runtime wasm file is
+//! compiled:
+//!`chain-spec-builder create --relay-chain <relay_chain_id> --para-id 1000 -r \
+//!     <path_to_template_wasm_file> named-preset development`
+//!
+//! Once the files are generated, you must export an environment variable called
+//! `RELAY_CHAIN_SPECS_DIR` which should point to the absolute path of the directory
+//! that holds the generated chain specs. The chain specs file names should be
+//! `minimal_chain_spec.json` for minimal and `parachain_chain_spec.json` for parachain
+//! templates.
+//!
+//! To start all tests here we should run:
+//! `cargo test -p template-zombienet-tests --features zombienet`
 
 #[cfg(feature = "zombienet")]
 mod smoke {
+	use std::path::PathBuf;
+
 	use anyhow::anyhow;
 	use zombienet_sdk::{NetworkConfig, NetworkConfigBuilder, NetworkConfigExt};
 
-	pub fn get_config(cmd: &str, para_cmd: Option<&str>) -> Result<NetworkConfig, anyhow::Error> {
-		let chain = if cmd == "polkadot" { "rococo-local" } else { "dev" };
+	const RELAY_CHAIN_SPECS_DIR_PATH: &str = "RELAY_CHAIN_SPECS_DIR";
+	const PARACHAIN_ID: u32 = 1000;
+
+	#[inline]
+	fn expect_env_var(var_name: &str) -> String {
+		std::env::var(var_name).expect("to have an environment variable set")
```

Comment:
We have this [qed pattern ](https://en.wikipedia.org/wiki/Q.E.D.) used everywhere across the codebase when `expect` is used. I would say it is also applicable here.
```suggestion
		std::env::var(var_name).expect(format!("{RELAY_CHAIN_SPECS_DIR_PATH} environment variable is set. qed.")
```

Reply:
Done here: [3486884](https://github.com/paritytech/polkadot-sdk/pull/5954/commits/348688497cf97a22119e4704221a2c4edf851a98)